Updated readme and makefile (#221)
kyma-project/eventing-manager: README.md

You need a Kubernetes cluster to run against. You can use [k3d](https://k3d.io/) to get a local cluster for testing, or run against a remote cluster.
> **Note:** Your controller automatically uses the current context in your kubeconfig file, that is, whatever cluster `kubectl cluster-info` shows.

## Install

1. To install the latest version of the Eventing manager on your cluster, run:

   ```bash
   kubectl apply -f https://github.com/kyma-project/eventing-manager/releases/latest/download/eventing-manager.yaml
   ```

> 2. To install the latest version of the default Eventing CR on your cluster, run:

Reviewer: When you first mention 'custom resource', use the full form (use lowercase for that) and the abbreviation in brackets (CR). Then, you can use the abbreviation throughout the doc.

Author: okay, I will try to remember it :)

         eventTypePrefix: "sap.kyma.custom"
   ```

### Undeploy Eventing Manager

Undeploy the Eventing Manager from the cluster:

   ```sh
   make undeploy
   ```

> ### Undeploy Eventing Manager

Author: I have updated it.

   kyma alpha deploy
   ```

2. Apply the Eventing module template to the Kubernetes cluster:

   > **NOTE:** You can get the latest released [module template](https://github.com/kyma-project/eventing-manager/releases/latest/download/module-template.yaml), or you can use the module template from the artifacts of `eventing-module-build` job either from the `main` branch or from your pull request.

   ```sh
   kubectl apply -f module-template.yaml
   ```

3. Enable the Eventing module:

   ```sh
   kyma alpha enable module eventing -c fast -n kyma-system
   ```

4. If you want to verify whether your Eventing module is deployed properly, perform the following checks:

   - Check if the Eventing resource has the ready state:

     ```shell
     kubectl get -n kyma-system eventing
     ```

   - Check if the Kyma resource has the ready state:

     ```shell
     kubectl get -n kyma-system kyma
     ```

### Uninstall Eventing Manager module with [Kyma Lifecycle Manager](https://github.com/kyma-project/lifecycle-manager/tree/main)

1. Delete Eventing Custom Resource (CR) from the Kubernetes cluster (if exists):

    ```sh
    kubectl delete -n kyma-system eventing eventing
    ```

> 2. Disable the NATS module:

Reviewer 2: ```suggestion
2. Disable the Eventing module:
```

Author: updated

   ```sh
   kyma alpha disable module eventing
   ```

3. Delete the Eventing module template:

   ```sh
   kubectl get moduletemplates -A
   kubectl delete moduletemplate -n <NAMESPACE> <NAME>
   ```

4. Check whether your Eventing module is uninstalled properly:

>    - Make sure that the Eventing Custom Resource (CR) does not exist. If it exists, then check the status of Eventing CR:

Reviewer: ```suggestion
   - Make sure that the Eventing CR does not exist. If it exists, check its status:
```

>    - Make sure that the Eventing Custom Resource (CR) does not exist. If it exists, then check the status of NATS CR:

Reviewer 2: why `check the status of NATS CR`?

Author: updated
